tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -93,3 +93,36 @@ Basic Whitelisting
     Should Contain  ${output}  Access denied to unauthorized registry
 
     Cleanup VIC Appliance On Test Server
+
+Configure Registry CA
+    # Install VCH without registry CA
+    Remove Environment Variable  GOVC_USERNAME
+    Remove Environment Variable  GOVC_PASSWORD
+    ${output}=  Install VIC Appliance To Test Server  vol=default
+
+    Should Not Contain  ${output}  Secure registry %{HTTPS_HARBOR_IP} confirmed
+
+    # Try to login to the HTTPS registry (should fail)
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} login -u admin -p %{TEST_PASSWORD} %{HTTPS_HARBOR_IP}
+    Should Not Contain  ${output}  Succeeded
+
+    # Add the HTTPS registry CA to cert pool with vic-machine configure
+    ${output}=  Run  bin/vic-machine-linux configure --target %{TEST_URL} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE} --name %{VCH-NAME} --registry-ca=./ca.crt --thumbprint=%{TEST_THUMBPRINT} --debug=1
+
```

Comment:
might check if reconfigure succeed here? to have more accurate error message if test failed